phpunit.xml

```diff
@@ -92,5 +92,10 @@
       <directory>src/plugins/orangehrmRecruitmentPlugin/test/Api</directory>
       <directory>src/plugins/orangehrmRecruitmentPlugin/test/Entity</directory>
     </testsuite>
+    <testsuite name="Directory">
+      <directory>src/plugins/orangehrmPlugin/test/Service</directory>
```

Comment:
```suggestion
```
please delete this line to avoid unit test failure

Reply:
Done